### PR TITLE
Add ORML pallets to parachain runtime

### LINKED ISSUE #47: Add ORML pallets to the parachain runtime
ORML crates to be added:
- [x] Traits
- [x] Tokens
- [x] Currencies

### Acceptance criteria:
Parachain contains Tokens & Currencies extrinsics to be called.

### How to test
Open polkadot.js app and connect to parachain (address to be specified).
Navigate to Extrinsics view.
Tokens & Currencies should be listed in available pallets and extrinsics for them should be present in extrinsics list.

### PR DESCRIPTION
**Pull request by vasylenko-yevhen**
Parachain runtime changes:
- Adds `primitives`;
- Adds `orml` crates: `traits`, `tokens`, `currencies`;
- Adds `Tokens` and `Currencies` pallets to the runtime.

Closes #47

### CODE REVIEW

File: bin/zero-parachain/node/src/chain_spec.rs [hunk 1]
```diff
@@ -194,7 +199,7 @@ fn testnet_genesis(
 		parachain_info: parachain_subzero_runtime::ParachainInfoConfig { parachain_id: id },
 		collator_selection: parachain_subzero_runtime::CollatorSelectionConfig {
 			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
-			candidacy_bond: EXISTENTIAL_DEPOSIT * 16,
+			candidacy_bond: cent(ZERO) * 16,
```
Comment (vovacha) [inline bin/zero-parachain/node/src/chain_spec.rs:202]: Maybe not hardcoding it. It's already assigned at runtime scope - `pub ExistentialDeposits`.
- `ExistentialDeposits::get(&ZERO)`
- Import `EXISTENTIAL_DEPOSIT` variable

Comment (vasylenko-yevhen) [reply bin/zero-parachain/node/src/chain_spec.rs:202]: Doing so will require ORML as a dependency for `node` crate. Not sure if it is a good idea.

Comment (vovacha) [reply bin/zero-parachain/node/src/chain_spec.rs:202]: I tend to disagree that hardcoding is better than having an additional dependency. 
On the other side, `candidacy_bond` and `EXISTENTIAL_DEPOSIT` connection is unclear, so I think we can leave it as is.

**Review by vovacha — APPROVED** [review]
LGTM

**Pull request merged**